Sources/SnowManLogic/ViewLogic/MockView.swift

```diff
@@ -0,0 +1,40 @@
+public class MockView: Viewable {
+	var ui: Receivable 
+	var guess: String!
```

Comment:
Optional properties on a instance are a smell to me. Not all smells mean that something's wrong about it's implementation, but they warrant consideration. Is this necessary here, or do you think there is a better way to handle this value?